### DISCUSSION TITLE
err: "Invalid UTF-8 stream “ when exporting .md to docx

### DISCUSSION THREAD

**Discussion by ashley681** [retorquere/zotero-better-bibtex#2528] [Q&A]
Hi everyone, I have problems exporting my .md documents to .docx with pandoc Plugin in obsidian. Do anybody know how to fix it?

Here is the  environment:
Zotero 6.0.26
Better BibTex for Zotero 6.7.79
Obsidian v1.3.4
Pandoc Plugin (for obsidian)  
Pandoc App
The filtering  lua script was loaded ( https://retorque.re/zotero-better-bibtex/exporting/zotero.lua)

Here is setting and the err
![image](https://github.com/retorquere/zotero-better-bibtex/assets/128587617/5c084db8-57ab-4373-adbd-201b717f5382)
![image](https://github.com/retorquere/zotero-better-bibtex/assets/128587617/40c44014-dc25-42e9-925d-09dc86b54e22)

Any suggestion will be appreciated!

**Comment by retorquere** [comment]
Everyone is me.

It could be that the input document is not utf-8. You'll have to ask obsidian, it is them that are supplying pandoc with its input.

**Reply by ashley681** [reply]
Thank u very much!  I thought you may know how to fix it because the lua script was provided in BBT support files, ( https://retorque.re/zotero-better-bibtex/exporting/zotero.lua).  I'll ask them anyway, thanks again!

**Reply by retorquere** [reply]
Is it complaining about the filter? Or about the source doc? Does the error go away if you remove the filter call?

**Reply by ashley681** [reply]
After removing the filter, I can get a docx but all citations turn to be plain text and cannot generate references list.
When using the filter, I get the error as I said before.

**Reply by retorquere** [reply]
Zotero.lua is utf-8 though, so I don't know where this error comes from. What happens if you run the command yourself from the command line?
